Source/EasyNetQ.Tests/ExchangeQueueBindingTests.cs

```diff
@@ -243,7 +252,8 @@ public void Should_declare_an_exchange()
             Arg.Is("direct"),
             Arg.Is(false),
             Arg.Is(true),
-            Arg.Any<IDictionary<string, object>>());
+            Arg.Any<IDictionary<string, object>>()
```

Comment:
why not to check WithExchangeAlternate as in tests above?

Reply:
Done